julia-mode.el

```diff
@@ -907,6 +907,7 @@ strings."
 (puthash "\\gamma" "γ" julia-latexsubs)
 (puthash "\\delta" "δ" julia-latexsubs)
 (puthash "\\upepsilon" "ε" julia-latexsubs)
+(puthash "\\varepsilon" "ε" julia-latexsubs)
```

Comment:
I'm not sure if it's just me but this renders differently from what I see locally....

![spectacle x20264](https://user-images.githubusercontent.com/712232/30490609-729e68be-9a08-11e7-92e0-0a279a7f401f.png)
![spectacle l20315](https://user-images.githubusercontent.com/712232/30490654-9bf7f1e4-9a08-11e7-98fb-9e0029e90628.png)

Reply:
The previous one renders the way I expect it to in the browser though...

![spectacle x20422](https://user-images.githubusercontent.com/712232/30490697-c4f18998-9a08-11e7-871c-ed659b5e76aa.png)